### ISSUE TITLE
SVG stroke-dasharray missing

### ISSUE THREAD

**Issue by DancingHedgehog** [ultralight-ux/Ultralight#416]
Hello from Mutate!
We thought of displaying ability cooldowns by using a pie chart that would fill up in relation to % time remaining.

To realize this we utilized SVG with stroke-dasharray, which looks fine in chrome:

![piechart_chrome](https://user-images.githubusercontent.com/90575500/162156371-cea389f2-2dd9-4ea1-afb7-401aa2204330.png)

```html
<html>
<head>
	<style>
	circle {
		fill: #655;
		stroke: yellowgreen;
		stroke-width: 50;
		stroke-dasharray: 60 158; /* 2π × 25 ≈ 158 */
	}
	svg {
		transform: rotate(-90deg);
		background: #655;
		border-radius: 50%;
	}
</style>
</head>
	<body>
		<div>
			<svg width="100" height="100">
				<circle r="25" cx="50" cy="50" />
			</svg>
		</div>
	</body>
</html>
```

But now in our engine using Ultralight, there is no visual representation of stroke-dasharray:
_Ultralight version: Latest nightly build available today (on 07/04/2022)._

![piechart_ultralight](https://user-images.githubusercontent.com/90575500/162155610-1d75d8f9-a6c0-4e9a-b1cc-e4538a7f6f09.png)

Please start supporting stroke-dasharray!

Jens
Mutate

**Comment by kboniadi** [comment]
@DancingHedgehog were you ever able to find a workaround for this? It still seems like this not natively supported in ultralight.

**Comment by adamjs** [comment]
It's supported now, should be landing on 1.3-trunk early next week.

**Comment by kboniadi** [comment]
@adamjs sorry for the ping. Are we able to access unstable/pre-release builds of the framework? For example, the one you just mentioned?

**Comment by adamjs** [comment]
Yes you can always access latest builds at: <https://github.com/ultralight-ux/Ultralight#getting-the-latest-sdk>

Those bins are auto-generated by the buildbots about 45 mins after each commit to the top-level `ultralight-ux/Ultralight` repo.

I'm cleaning up a few rendering issues for 1.3 now and will likely update Monday.
